src/Compilers/Core/Portable/DiagnosticAnalyzer/DiagnosticAnalysisContext.cs

```diff
@@ -161,6 +171,11 @@ public void RegisterOperationAction(Action<OperationAnalysisContext> action, par
             this.RegisterOperationAction(action, operationKinds.AsImmutableOrEmpty());
         }
 
+        internal void RegisterOperationActionParamsArrayInternal(Action<OperationAnalysisContext> action, params OperationKind[] operationKinds)
```

Comment:
Because this is a temporary solution and those methods are internal, I chose to make it easier to get the intended method through reflection, rather than a consistent and clean interface. But if consistency is preferred in this case, i will fix it.